src/ussd_registration.js

```diff
@@ -292,14 +295,13 @@ go.app = function() {
         // PaginatedChoiceState st-12 & 13
         self.add('state_baby_birth_month_year', function(name) {
             var today = go.utils.get_today(self.im.config);
-            var start_month = today.month();
             return new PaginatedChoiceState(name, {
                 question: $(questions[name]),
                 characters_per_page: 182,
-                options_per_page: null,
+                //options_per_page: null,
```

Comment:
Ok. You'll notice it changed the pagination - that's because the default is 8 if you don't specify `null` (in case you were curious).